src/codec-protobuf.js

```diff
@@ -11,36 +11,45 @@ protobuf.configure();
 
 const { Root } = protobuf;
 
-const AnyType = Root.fromJSON(JSON.parse(schema)).lookupType('codecprotobuf.AnyType');
+const Any = Root.fromJSON(JSON.parse(schema)).lookupType('dxos.codec.Any');
 
+/**
+ * Encodes and decodes protocol buffer messages.
+ */
 class Codec {
   constructor (options = {}) {
     const { verify = false, decodeWithType = true } = options;
 
+    // TODO(burdon): Rename strict?
     this._verify = verify;
```

Comment:
protobufjs call this operation `verify`, is it necessary to change their naming convention?